scripts/validate_record.py

```diff
@@ -0,0 +1,290 @@
+import requests
+import json
+import os
+import sys
+from typing import Dict, List
+import utils
+from misc import from_lower_camel_case
+
+
+logger = utils.create_logging_instance("validate_record")
+
+
+class validate_record():
+    def __init__(self, record_type: str, records: Dict, rulesets: List, batch_size: int = 600):
```

Comment:
Please add class description and documentation for all methods inside it

Reply:
done in b536f32